website_breadcrumb/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Website Breadcrumbs
+===================
+
+This module allows you to have breadcrumbs in any page of your website.
+
```

Comment:
If an URL is not present in a menu (as indicated in the **Configuration** section), no breadcrumbs will display.

Definitely worse than https://github.com/OCA/website-cms/pull/1, but much smaller too.